### ISSUE TITLE
[Enabler] Example showing how to concatenate number of commands in zos_tso_command

### ISSUE THREAD

**Issue by EdwardMcCarthy** [ansible-collections/ibm_zos_core#777]
### Is there an existing issue for this?

- [X] There are no existing issues.

### Ansible module

zos_tso_command

### Enabler description

I'd like to suggest adding an example to 

https://github.com/IBM/z_ansible_collections_samples

to show an example of how to concatenate a series of TSO commands when using zos_tso_command

If use this to try to copy a module from one PDS to another:

    - name: Use TSO to copy updated module into CICS
      zos_tso_command:
        commands:
          - ALLOCATE DDNAME(IN1) DSNAME('IBMUSER.DBB.CICSIVP.LOAD(EGUIMENU)') SHR; ALLOCATE DDNAME(OUT1) DSNAME('EDMCAR.CICS.COBOL.LOADLIB(EGUIMENU)') SHR; OCOPY INDD(IN1) OUTDD(OUT1) BINARY
          - ALLOCATE DDNAME(OUT1) DSNAME('IBMUSER.CICS.COBOL.LOADLIB(EGUIMENU)') SHR
          - OCOPY INDD(IN1) OUTDD(OUT1) BINARY
      register: tcpResult

the copy fails, because each of the commands runs in own env, all cmds work ok, but for the last command, the allocations are not there

This playbook example shows two ways to get around the above:

  vars:
    #Folded style '>', eg fold new lines '\n' into spaces with a chomp mode
    # strip '-' that will strip all newlines including traling ones. Combine
    # them to make '>-'
    cmd: >-
      ALLOCATE DDNAME(IN1) DSNAME('IBMUSER.DBB.CICSIVP.LOAD(EGUIMENU)') SHR;
      ALLOCATE DDNAME(OUT1) DSNAME('IBMUSER.CICS.COBOL.LOADLIB(EGUIMENU)') SHR;
      OCOPY INDD(IN1) OUTDD(OUT1) BINARY

  tasks:
    - name: Result cmd
      debug:
        msg: "{{ cmd }}"

    - name: Use TSO to copy updated module into CICS using cmd var.
      zos_tso_command:
        commands:
          "{{ cmd }}"
      register: tcpResult

    - name: Result tcpResult
      debug:
        msg: "{{ tcpResult }}"
  
    - name: Use TSO to copy updated module into CICS using flow sclars
      zos_tso_command:
        commands: >-
          ALLOCATE DDNAME(IN1) DSNAME('IBMUSER.DBB.CICSIVP.LOAD(EGUIMENU)') SHR;
          ALLOCATE DDNAME(OUT1) DSNAME('IBMUSER.CICS.COBOL.LOADLIB(EGUIMENU)') SHR;
          OCOPY INDD(IN1) OUTDD(OUT1) BINARY
      register: tcpResult

    - name: Result tcpResult
      debug:
        msg: "{{ tcpResult }}"

This could be useful example to have in the above samples link

**Comment by ddimatos** [comment]
I agree @EdwardMcCarthy and thank you for taking the time to open this, this was a nice to work on this with you.

I still have to question if the `zos_tso_command` should support this natively. The only thing that comes to mind would be an option in the module for lack of a better name, eg `concatenate=True` such we concatenate commands with semi-colons and run it as a single command. 

For now the folding scalar is a good addition.
